src/status_im2/contexts/profile/login/events.cljs

```diff
@@ -175,7 +190,7 @@
      cofx
      {:db (assoc-in db [:profile/login :password] password)}
      (navigation/init-root :progress)
-     (login))))
+     (biometry-login))))
```

Comment:
is biometry appropriate here? should it not be biometrics-login ? 
It kind of makes sense but never really heard of this domain referred to as "biometry"

Reply:
I was using them interchangeably till now, but they seem to be different :open_mouth: